cmd/kms-rest/startcmd/start_test.go

```diff
@@ -29,13 +29,13 @@ const (
 
```

Comment:
I think the GetTLS method is what's keeping your code coverage low... Take a look at the unit tests in the EDV project - you should be able to do the same thing here to get your code coverage up

Reply:
Thank you Derek for that suggestion. For some reason, I didn't realize that just adding the flag with an empty value can trigger needed failure behavior for that flag. So your example code was very helpful here.